pyproject.toml

```diff
@@ -14,20 +17,26 @@ log_cli_level = "INFO"
 line-length = 99
 target-version = ["py38"]
 
-[tool.isort]
-line_length = 99
-profile = "black"
-
 # Linting tools configuration
-[tool.flake8]
-max-line-length = 99
-max-doc-length = 99
+[tool.ruff]
+line-length = 99
+select = ["E", "W", "F", "C", "N", "D", "I001"]
+extend-ignore = [
+    "D203",
+    "D204",
+    "D213",
+    "D215",
+    "D400",
```

Comment:
my personal opinion: it can be disabled

Reply:
The convention that we've used throughout our other projects is that the summary line for a docstring is an imperative complete sentence. It would be best to keep the consistency.

Reply:
yeah, @dusktreader, can you give a little more explanation here?

Reply:
@jamesbeedy I updated the original comment to add more context.